traffic_ops/testing/api/v3/jobs_test.go

```diff
@@ -82,6 +83,28 @@ func CreateTestInvalidationJobs(t *testing.T) {
 	}
 }
 
+func CreateInvalidJob(t *testing.T) {
```

Comment:
To follow naming conventions, this should be `CreateTestInvalidJob()`.

Reply:
done in bae7ca548739e0d9fe501afba8b2c2559b7843f9